umea/meetup.yaml

```diff
@@ -0,0 +1,82 @@
+city: Umeå
+country: sweden
+meetupID: Cloud-Native-Northern-Sweden
+meetups:
+- address: Tvistevägen 47
+  attendees: 19
+  date: "2018-05-28T17:00:00Z"
+  duration: 2h0m0s
+  id: 261450080
+  name: Tremds in cloud native - a summary from KubeCon + CloudNativeCon Barcelona
```

Comment:
typo: `trends`

Reply:
ah right this is probably coming directly from the meetup API and can't be changed... ok